etc/cvmfs/config.d/ligo.osgstorage.org.conf

```diff
@@ -1,5 +1,5 @@
 CVMFS_EXTERNAL_URL="https://stashcache.t2.ucsd.edu:8444//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;https://dtn2-daejeon.kreonet.net:8444//user/ligo/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://fiona.uvalight.net:8444//user/ligo/;https://stashcache.gravity.cf.ac.uk:8444//user/ligo/;https://xcache.cr.cnaf.infn.it:8443//user/ligo/;https://xcachevirgo.pic.es:8443//user/ligo/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://fiona.uvalight.net:8444//user/ligo/;https://stashcache.gravity.cf.ac.uk:8444//user/ligo/;https://xcache.cr.cnaf.infn.it:8443//user/ligo/;https://xcachevirgo.pic.es:8443//user/ligo/;https://ligo.hpc.swin.edu.au:8443//user/ligo/"
```

Comment:
Actually I'd prefer this one to also be on the previous line because that's the convention I was following, although it wasn't documented.  For that purpose, please add a comment before the first line that says "# subset of stashcache servers configured by OSG" and another comment before the second line that says "# stashcache servers not configured by OSG"